site2/docs/about.md

```diff
@@ -19,7 +19,7 @@ This portal holds a variety of topics, tutorials, guides, and reference material
 Select one of the content blocks below to begin your Pulsar journey. If you ...
 * Are new to Pulsar, start with **About Pulsar** to learn about features and concepts.
```

Comment:
How about adding links for them so that users can jump there more quickly?
<img width="883" alt="image" src="https://user-images.githubusercontent.com/50226895/188040435-87d558c5-87e4-45ea-a7e4-f0ed52676ba6.png">

Reply:
Good suggestion but I think it's out of the scope of this PR. It can be covered in https://github.com/apache/pulsar/pull/17284.